Doppler.HtmlEditorApi/Utils.cs

```diff
@@ -17,4 +17,23 @@ public static IEnumerable<T> EmptyIfNull<T>(this IEnumerable<T> source)
 
     public static string FallbackIfNullOrEmpty(this string source, string fallback)
         => string.IsNullOrEmpty(source) ? fallback : source;
+
```

Comment:
is needed this function?

Reply:
No, it is not needed. It was made to analyze the meta object to find a value, in this case for a dynamic component